<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>621</width>
    <height>457</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Pocsel Plugin Packager</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Plugin root directory:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLineEdit" name="pluginRootLineEdit"/>
        </item>
        <item>
         <widget class="QPushButton" name="pluginRootPushButton">
          <property name="text">
           <string>Browse...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Package in file:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLineEdit" name="destFileLineEdit"/>
        </item>
        <item>
         <widget class="QPushButton" name="destFilePushButton">
          <property name="text">
           <string>Browse...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="packagePushButton">
        <property name="text">
         <string>Package plugin</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QCheckBox" name="replaceCheckBox">
        <property name="text">
         <string>Replace existing plugin without asking</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Packaging result:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="logPlainTextEdit">
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
      <property name="tabChangesFocus">
       <bool>true</bool>
      </property>
      <property name="undoRedoEnabled">
       <bool>false</bool>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="plainText">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>pluginRootLineEdit</tabstop>
  <tabstop>pluginRootPushButton</tabstop>
  <tabstop>destFileLineEdit</tabstop>
  <tabstop>destFilePushButton</tabstop>
  <tabstop>packagePushButton</tabstop>
  <tabstop>replaceCheckBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
